application/web/index.html

```diff
@@ -42,7 +42,18 @@
                         <label class="new_entry">Create New Entry</label>
                     </div>
                 </button>
-
+                <div class = "home-list">
+                    <div class="home-entry-descriptor">
+                        <span>Entry Name</span>
+                        <span>Date Logged</span>
+                    </div>
+                    <button class="home-single-entry">
+                        <span class="home-entry-name">My Journal</span>
+                        <span>Date Created</span>
+                        <img src="/application/web/assets/vdots-entry-black.png">
```

Comment:
I think this might be correct path: 
<img src="../web/assets/vdots-entry-black.png">

but you need to add some styling. I think you can follow the same styling as the 3 black dots in the entry page